src/BotPlutusInterface/CardanoCLI.hs

```diff
@@ -70,31 +67,19 @@ import Ledger.Interval (
  )
 import Ledger.Scripts (Datum, DatumHash (..))
 import Ledger.Scripts qualified as Scripts
-import Ledger.Tx (
-  ChainIndexTxOut,
-  RedeemerPtr (..),
-  Redeemers,
-  ScriptTag (..),
-  Tx (..),
-  TxIn (..),
-  TxInType (..),
-  TxOut (..),
-  TxOutRef (..),
-  txId,
- )
-import Ledger.TxId (TxId (..))
+import Ledger.Tx (RedeemerPtr (..), Redeemers, ScriptTag (..), Tx (..), TxId (..), TxIn (..), TxInType (..), TxOut (..), TxOutRef (..), txId)
+import Ledger.Tx.CardanoAPI (toCardanoAddressInEra)
```

Comment:
nit; Not sure if you care, but you generally seem to use explicit imports everywhere but here